src/scripts/underground/Underground.ts

```diff
@@ -58,6 +58,30 @@ class Underground implements Feature {
                     GameHelper.createArray(50, 250, 50), GameConstants.Currency.diamond),
                 GameHelper.createArray(0, 10, 2)
             ),
+            new UndergroundUpgrade(
+                UndergroundUpgrade.Upgrades.SurveyCharge_Cost, 'Survey Charge Cost', 5,
+                AmountFactory.createArray(
+                    GameHelper.createArray(50, 250, 50), GameConstants.Currency.diamond),
+                GameHelper.createArray(0, 5, 1)
+            ),
+            new UndergroundUpgrade(
+                UndergroundUpgrade.Upgrades.SurveyCharge_Efficiency, 'Survey Charge Efficiency', 4,
+                AmountFactory.createArray(
+                    GameHelper.createArray(100, 400, 100), GameConstants.Currency.diamond),
+                GameHelper.createArray(0, 4, 1)
+            ),
+            new UndergroundUpgrade(
+                UndergroundUpgrade.Upgrades.NewYLayer, 'Larger underground, +1 Max Item', 1,
```

Comment:
I don't think we need to increase the size of the Underground, this seems to put you at more of a disadvantage than anything else, also feels quite pointless as a 1 time upgrade.

Reply:
Due to the way items are added to the Underground, this upgrade essentially serves to make it easier to add items. You more consistently get higher item counts so it's actually more of a min item upgrade than max. 

The reason it's a 1 time upgrade is because anything more than that causes issues with the underground size. I don't think it's pointless though; point being it's an endgame upgrade when you've got everything else and can afford the energy requirements to mine out more tiles. 

It's why the price of it is so high, I recognise for new players this would be a disadvantage but not in the late game when you've got the max items upgraded already but not the space to spawn required them.